### DISCUSSION TITLE
I need help to skip the setup menu (GoPro Hero 9 Without Screen)

### DISCUSSION THREAD

**Discussion by gustavokklein** [gopro/labs#868] [Q&A]
Hey guys,
I have a GoPro 9 and I have a serious problem. One day my rear screen stopped working, I tried to buy a new one and replace it, but it didn't work (I believe it was a problem with the mainboard).

Since then, I've been using GoPro Labs to set up my camera for everything.

However, a few days ago I started having problems with crashes, and I did a factory reset on it. Everything is back to normal (or not) because I can't use it anymore, it's stuck in initial configuration mode, and the front screen keeps showing the "GoPro 9 Black" logo.

I was even able to set commands with Labs and connect it to my phone, but I can't get it out of this configuration mode in any way. Does anyone know of a command to do this, or any other way to skip this mode (Remembering that I don't have a working rear screen).

I would really appreciate it if anyone can help!!

**Comment by dnewman-gpsw** [comment]
The problem is you did a factory reset, which requires a working screen, unless it is completely removed.  Try scanning !MNAKE=1, this emulates a screen disconnect.
